<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="QA-2-Win8_1-64" name="com.automation.main.TC15654TryToMoveRecordingWhileItIsBeingMoved" tests="2" failures="0" timestamp="5 Sep 2016 08:40:35 GMT" time="123.542" errors="1">
  <testcase name="loadPage" time="6.237" classname="com.automation.main.TC15654TryToMoveRecordingWhileItIsBeingMoved"/>
  <testcase name="loginCourses" time="117.305" classname="com.automation.main.TC15654TryToMoveRecordingWhileItIsBeingMoved">
    <error type="junit.framework.AssertionFailedError" message="timeout">
      <![CDATA[junit.framework.AssertionFailedError: timeout
	at junit.framework.Assert.fail(Assert.java:50)
	at com.automation.main.LoginHelperPage.loginCourses(LoginHelperPage.java:141)
	at com.automation.main.TC15654TryToMoveRecordingWhileItIsBeingMoved.loginCourses(TC15654TryToMoveRecordingWhileItIsBeingMoved.java:108)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:207)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:178)
]]>
    </error>
  </testcase> <!-- loginCourses -->
</testsuite> <!-- com.automation.main.TC15654TryToMoveRecordingWhileItIsBeingMoved -->
